node_hapi/app/index.js

```diff
@@ -38,4 +38,13 @@ server.route({
     }
 });
 
+server.route({
+    method: 'DELETE',
+    path: '/api/clients/{id}',
+    handler: async (request, reply) => {
+        const rowsDeleted = await knex('clients').where('id', request.params.id).del();
+        return rowsDeleted ? reply().code(204) : reply().code(404);
```

Comment:
@mradamh Just a brief note here, which you can certainly choose to ignore. The sad path is currently completely untested. If you wanted to add a test to verify this behavior I wouldn't be upset about it :)